### ISSUE TITLE
Server does not work when UDF with same function defined as global and local is registered.

### ISSUE THREAD

**Issue by sacheendra** [aerospike/aerospike-server#38]
Registering a lua udf with 2 function of the same name, one local and one global, causes problems with the server. 
Example:
local function myfunc()
end
function myfunc()
end

I am using aerospike from the docker image "aerospike/aerospike-server"

**Comment by psi-mankoski** [comment]
Sacheendra --
Since only the Aerospike server itself can directly call global scope functions, it is necessary to use an intermediary local scope variable (or local function) to invoke the global scope function. Other than that, there is no problem having local and global definitions a given name within a file.

**Comment by sacheendra** [comment]
Yeah, calling a global function does not work, which is mentioned in the docs. But in the above situation, the server goes into an **infinite loop** in the above situation which should not happen.
